### PR TITLE
Porting language syllables

### PR DESCRIPTION
**Pull request by Jamini**
Mob Syllable port from bay. Converts languages you do not understand into randomized syllable strings.

E.G.

Nasir Khayyam mrowls, "Rreshi, roxa'mi'tok"
Skrellian Dubstep Champion warbles, "wubdubwrashiwubwubdub"

Tested and confirmed working with scopes. Kept functionality of sign language. 
Also contains code for emote language ( ! ) allowing radio emotes with ;!

### CODE REVIEW

File: code/modules/mob/living/carbon/human/say.dm [hunk 1]
```diff
@@ -123,27 +129,14 @@
...
-			if(!(stunned >= 4))
-				if(message_mode)
-					if(message_mode in (radiochannels | "department" | "special"))
-						if(l_ear && istype(l_ear,/obj/item/device/radio))
-							l_ear.talk_into(src,message, message_mode, verb, speaking)
-							used_radios += l_ear
-						else if(r_ear && istype(r_ear,/obj/item/device/radio))
-							r_ear.talk_into(src,message, message_mode, verb, speaking)
-							used_radios += r_ear
+			if(message_mode)
```
Comment (SoundScopes) [inline code/modules/mob/living/carbon/human/say.dm]: if(!(stunned >= 4))

Comment (Jamini) [inline code/modules/mob/living/carbon/human/say.dm]: Ah, missed that one. Re-adding it now.

There should be a more elegant way to implement "No radios while stunned" than having an if statement in each switch case though. Hmm

Comment (SoundScopes) [inline code/modules/mob/living/carbon/human/say.dm]: There probably is, I remember seeing that this was the old way to do it.

All looks good to me.

**Comment by Jamini** [comment]
Rebased and removed unneccesary commits. Should be only the files needed now

**Pull request merged**